arch/arm/soc/st_stm32/stm32f2/Kconfig.defconfig.stm32f215xx

```diff
@@ -0,0 +1,18 @@
+# Kconfig - ST Microelectronics stm32f215xx MCU
+#
+# Copyright (c) 2018 qianfan Zhao <qianfanguijin@163.com>
```

Comment:
Provide your own copyright.
Even if this is copy/paste, you're the one doing the test and upstream effort.